src/XamlStyler/Options/StylerOptions.cs

```diff
@@ -298,6 +298,14 @@ public StylerOptions(string config = "")
         [JsonIgnore]
         public bool SearchToDriveRoot { get; set; }
 
+        [Category("Attribute Reordering")]
+        [DisplayName("Ignore Design-Time Reference Prefix")]
+        [JsonProperty("IgnoreDesignTimeReferencePrefix", DefaultValueHandling = DefaultValueHandling.Populate)]
+        [Description("Defines whether attributes with the design-time reference namespace prefix are ignored when ordering.")]
+        [DefaultValue(false)]
+        [TypeConverter(typeof(BooleanConverter))]
```

Comment:
I don't think this TypeConverter is necessary.

Reply:
Ok, corrected